### PR TITLE
Adds output of X-OT-Backend from disco host.

### PR DESCRIPTION
**Pull request by pennello**
https://opentable.atlassian.net/browse/OTPL-1402

### CODE REVIEW

File: otpl-service-check [hunk 1]
```diff
@@ -115,8 +115,12 @@ class Main(object):
 
   def get_announcements(self):
     url = urljoin(self.args.discovery, 'state')
-    state = self.requestsget(url, discotimeout).json()
-    return [a for a in state if a['serviceType'] == self.args.service]
+    resp = self.requestsget(url, discotimeout)
+    if not resp.headers: backend = None
+    else: backend = resp.headers.get('X-OT-Backend') or None
```
Comment (stevenschlansker) [inline otpl-service-check]: what does the `or None` do here?  If it's not present, won't it already be `None`?

Comment (pennello) [inline otpl-service-check]: Correct.  But if it's `''`, then I'd like us to output `None`, still.  Same deal with testing `resp.headers`.  `None` or empty, we set `backend` to `None`.

**Comment by pennello** [comment]
Ok @stevenschlansker, let me know what you think of updating it to print just `X-OT-Backend-Task-Host` (454753db7f7f21ba85fe2998d21da7cff0067348).

**Comment by pennello** [comment]
Just gonna go ahead with this.

**Pull request merged**